packages/api/src/task/scheduler.ts

```diff
@@ -162,27 +162,6 @@ export class TaskScheduler {
         });
         await this.triggerRecordingReadyWebhook(task);
         break;
-      case "transcode":
```

Comment:
Can you also remove any references on the API schema (which generates all the model typings here as well) to the removed tasks? e.g. there's a `task.type` enum field which is allowed to be `transcode` or `import` today. We should remove that (even tho the old DB objects might have them, that's not a problem)